### PR TITLE
feat(button-group): button group styling and documentation

### PR DESCRIPTION
**Pull request by ArianneArcher**
FLO-11212

Button Groups - Since they work as a filter (toggle) Added  a pointer-events: none; (line 68) to the :active. Though was it would prevent the user from clicking on it when active and refreshing the active content. Let me know if this needs to change.

### CODE REVIEW

File: scss/_buttons.scss [hunk 1]
```diff
@@ -87,7 +87,6 @@ fieldset:disabled a.btn {
     background-color: $gray-400;
     border-color: $gray-400;
     opacity: 1;
-    pointer-events: none;
```
Comment (ArianneArcher) [inline scss/_buttons.scss:90]: Removed duplicate css

**Comment by bexux** [comment]
Do we have all of these button group features available for web app? Also, would we have a disabled button in a button group?

**Comment by ArianneArcher** [comment]
**Disabled State**

Button groups are used to filter content in a section into smaller sections. If content in a button group state was disabled then the content would not be available therefore a button group would not be used to filter the content.

**Comment by ArianneArcher** [comment]
The top two examples are in the design system, but the button groups under System Extras. I'll update the documentation to clarify. Thanks for catching that.

**Review by bexux — APPROVED** [review]

**Pull request merged**